nestkernel/communicator.cpp

```diff
@@ -146,7 +149,9 @@ nest::Communicator::init( int* argc, char** argv[] )
   // generate and commit struct
   MPI_Type_struct( 2, blockcounts, offsets, source_types, &MPI_OFFGRID_SPIKE );
   MPI_Type_commit( &MPI_OFFGRID_SPIKE );
-
+  // set up order of communication if using CPEX
+  if ( !use_Allgather_ )
+    init_communication();
   initialized_ = true;
```

Comment:
@tammoippen, @heplesser : it was not intended to reintroduce CPEX communication. I'm sorry that I didn't check this. In fact, these changes where made when CPEX communication was still implemented in the communicator. Sorry for the noise.